File: EducationManagement.Database/Models/Level.cs
using System;
using System.Collections.Generic;

namespace EducationManagement.Database.Models
{
    public partial class Level
    {
        public Level()
        {
            SubjectLevel = new HashSet<SubjectLevel>();
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public int? MinEntryTaskScore { get; set; }

        public ICollection<SubjectLevel> SubjectLevel { get; set; }
    }
}

File: EducationSystem.WEB/src/app/_models/Level.ts
export class Level {
    public Name: string;
    public Id: number;
    public MinEntryTaskScore: number;

    constructor(name?: string, id?: number, minEntryTaskScore?: number) {
        this.Name = name;
        this.Id = id;
        this.MinEntryTaskScore = minEntryTaskScore;
    }
}